src/markdown-pages/nr1-cli/nr1-nerdpack.mdx

```diff
@@ -0,0 +1,142 @@
+
+### Options
+| | |
+|---|---|
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+*** 
+
+## `nr1 nerdpack:uuid`
+### Show or generate your Nerdpack's UUID
+Prints the UUID (Universal Unique ID) of your Nerdpack, by default. The UUID determines what data the Nerdpack can access and who can subscribe to the Nerdpack. To deploy a Nerdpack you didn't make, you'll have to assign it a new UUID by using the `-g` or `--generate` option.
+
+For more details, see the [Access to data page](https://developer.newrelic.com/build-tools/new-relic-one-applications/guide-to-authentication--data-access--and-permissions#Appaccesstodata) or [our workshop](https://github.com/newrelic/nr1-workshop/blob/master/lab-cli/INSTRUCTIONS.md) on GitHub.
+
+### Usage
+`$ nr1 nerdpack:uuid` 
+
+### Options
+| | |
+|---|---|
+|  `--profile=PROFILE`| The authentication profile you want to use. |
+|  `-f, --force`       | If present, it will override the existing UUID without asking. |
+|  `-g, --generate`     | Generates a new UUID if not available. |
+|  `--verbose`        | Adds extra information to the output. |
+
+<br/>
+
+***
+
+## `nr1 nerdpack:publish`
+### Publish your Nerdpack
+Publishes your Nerdpack to New Relic. After publishing, you can use `nr1 nerdpack:deploy` to deploy it to a channel and make it available to users. 
+
+Please note:
+* If no additional parameters are passed in, this command will automatically deploy the Nerdpack onto the DEV channel. 
+* If you want to specify your own list of deploy channels, add the `--channel` option. For example, `$ nr1 nerdpack:publish --channel BETA --channel STABLE`.
+* If you want to disable this behavior, add `--skip-deploy` to the command. Then, you can use `nr1 nerdpack:deploy` to perform a deploy manually.
+
+For more on publishing and deploying, see [Deploy to New Relic One](/build-tools/new-relic-one-applications/publish-deploy).
+
+### Usage
+`$ nr1 nerdpack:publish` 
+
+### Options
+| | |
+|---|---|
+| `-B, --skip-build`                          | Skips the previous build process. |
+| `-D, --skip-deploy`                        | Skips the following deploy process. |
+| `-c, --channel=DEV/BETA/STABLE`            | Specifies the channel to deploys to. [default: STABLE] |
+|  `-f, --force`                              |  Forces the publish, overriding any existing version in the registry. |
+|  `--dry-run`                                 | Undergoes publishing process without actually publishing anything. |
+|  `--extra-metadata-path=extra-metadata-path` | Specifies a json file .path with extra metadata. [default: extra-metadata.json] |
+|  `--prerelease=STRING`                   | The value you enter will be appended to the current version of generated files. |
+|  `--profile=PROFILE`                      | The authentication profile you want to use. |
+|  `--verbose`        | Adds extra information to the output. |
```

Comment:
I've seen a lot of spacing issues with the markdown tables piping -  but I think it doesn't matter and should render correctly, may want to double check that.

Reply:
Ran the Markdown files on local host and saw no issues, but I'll go ahead and fix the spacing for convention purposes. Markdown tables aren't great.